src/server/config/config.js

```diff
@@ -188,7 +188,8 @@ module.exports = class Config {
   getSchema() {
     if (!this[schema]) {
       this[schema] = (function convertToSchema(children) {
-        let schema = Joi.object().keys().default();
+        let objKeys = zipObject([...children]);
```

Comment:
I don't think this is doing what you think it is. Key validation is added per key [here](https://github.com/elastic/kibana/pull/6554/commits/c72c440a1ccf80f4da7f2b0550b1d564809b65fc#diff-00313017c0120ff426b859f2dd5112cdR201)